python/ppml/example/pytorch_nn_lr/pytorch-nn-lr-tutorial.md

```diff
@@ -96,19 +96,30 @@ result = ppl.predict(x)
 ## 3 Run FGBoost
 FL Server is required before running any federated applications. Check [Start FL Server]() section for details.
 ### 3.1 Start FL Server in SGX
-// TODO: add this section after running FL Server in SGX succesfully in this example.
-
-Modify the config file `ppml-conf.yaml`
-```yaml
-# the port server gRPC uses
-serverPort: 8980
-
-# the number of clients in this federated learning application
-clientNum: 2
+#### 3.1.1 Start the container
+Before running FL Server in SGX, please prepaer keys and start the BigDL PPML container first. Check  [3.1 BigDL PPML Hello World](https://github.com/liu-shaojun/BigDL/tree/ppml_doc/ppml#31-bigdl-ppml-hello-world) for details.
+#### 3.1.2 Run start_fl_server.py
+First, enter the container.
+```bash
+docker exec -it YOUR_CONTAINER bash
+cd /ppml/trusted-big-data-ml
+```
+Then, write the following commands into a start-python-fl-server-sgx.sh.
+```bash
+#!/bin/bash
+cd /ppml/trusted-big-data-ml
+/graphene/Tools/argv_serializer bash -c " /opt/jdk8/bin/java     -cp '/ppml/trusted-big-data-ml/work/spark-3.1.2/conf/:/ppml/trusted-big-data-ml/work/spark-3.1.2/jars/*'     -Xmx10g org.apache.spark.deploy.SparkSubmit     --master 'local[4]'   start-fl-server.py -p 8980 -c 2" > /ppml/trusted-big-data-ml/secured-argvs
+./init.sh
+SGX=1 ./pal_loader bash 2>&1 | tee fl-server.log
+```
+You can set port with `-p` or `--port` and set client number with `-c` or `--client-num` while the default settings are `port=8980` and `client-num=2`.
+Then run the script,
+```bash
+bash start-python-fl-server-sgx.sh
 ```
-Then start the FL Server
+or run it outside of the container.
 ```bash
-python BigDL/python/ppml/src/bigdl/ppml/fl/nn/fl_server.py
+docker exec -it YOUR_CONTAINER bash start-python-fl-server-sgx.sh
```

Comment:
This script does not exist in this PR. Did you put it in docker image?

Reply:
The origin name of script in spark-k8s-client-attestation-devel-HY is start-python-flserver2-sgx.sh.
I've changed the name now